src/pal/inc/pal.h

```diff
@@ -447,12 +448,10 @@ BOOL
 PALAPI
 PAL_NotifyRuntimeStarted(VOID);
 
-#ifdef __APPLE__
```

Comment:
I would still prefer to keep the function ifdef-ed for `__APPLE__` only and call it conditionally in ProcessDescriptor ProcessDescriptor::FromCurrentProcess() instead.